docs/azure_jumpstart_ag/contoso_supermarket/freezer_monitor/_index.md

```diff
@@ -4,4 +4,261 @@ weight: 100
 toc_hide: true
 ---
 
-# Overview
\ No newline at end of file
+## Contoso Supermarket Freezer Monitor Overview
+
+### Overview
+
+Contoso Supermarket is obsessed with achieving the highest levels of food safety. To support this obsession Contoso has invested in technology to let it know when any food in a store's freezers is potentially unsafe due to the freezer reaching temperatures that would allow the food to thaw and pathogens to grow.
+
+Contoso has installed IoT sensors in each freezer in each store to detect both temperature and humidity. The IoT sensors send current measurements via Message Queuing Telemetry Transport (MQTT) to a broker service in each store. The broker service forwards the data to [Azure IoT Hub](https://azure.microsoft.com/products/iot-hub/) and along to [Azure Data Explorer (ADX)](https://azure.microsoft.com/products/data-explorer/) database for aggregation with data from all stores and analysis. In addition, the sensor data is also sent to a dashboard in each store for visualizing sensor history and to enable early warning notifications to the store manager when a safety issue is imminent.
+
+Contoso Supermarket is researching a number of additional health and safety systems that will leverage the same IoT infrastructure. These include:
+
+- Air quality sensors to detect the presence of smoke or other contaminants
+- Water quality sensors to detect the presence of contaminants in the water supply
+- Motion and presence sensors to lights should be turned on for personal safety
```

Comment:
```suggestion
Contoso Supermarket is researching a number of additional health and safety systems that will leverage the same IoT infrastructure. These include:

- Air quality sensors to detect the presence of smoke or other contaminants.
- Water quality sensors to detect the presence of contaminants in the water supply.
- Motion and presence sensors to lights should be turned on for personal safety.
```

Reply:
I would make a note that this is a future investment and out of scope for this one.

Reply:
@likamrat - The point of the research statement is to let the reader know that the same infra can be used for more sensors in the future, not that we plan to add those to this jumpstart release. Do you think readers will be expecting jumpstart to add it?

Also, those bullets are sentence fragments and shouldn't really have periods per guidance like this https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/punctuating-bullet-point-lists, but I'm flexible.